cv_next/server/api/firebaseHelper.ts

```diff
@@ -44,15 +44,18 @@ export default class FirebaseHelper {
      * Method initiates the firebase app if not initiated yet
      * @returns FirebaseApp instance
      */
-    public static getFirebaseInstance(): FirebaseApp {
+    public static getFirebaseInstance(): App {
+      
       if (
-        FirebaseHelper.firebaseAppInstance === null ||
-        FirebaseHelper.firebaseAppInstance === undefined
+        (FirebaseHelper.firebaseAppInstance === null ||
+        FirebaseHelper.firebaseAppInstance === undefined) 
       ) {
-        FirebaseHelper.firebaseAppInstance = initializeApp(
-          Definitions.FIREBASE_CONFIG
-        );
+        FirebaseHelper.firebaseAppInstance = (getApps().length === 0) ? initializeApp({
+          credential: credential.cert(cvSdk)
+        }, "cv-next") :
```

Comment:
Please use a constant for `"cv-next"` - perhaps change the project name in `package.json` to its proper name, and import it from there?
If you don't want to create a hard relationship between the project name and the firebase project, you can also save this name as a global constant instead (in a "constants" folder)